.travis.yml

```diff
@@ -13,3 +14,9 @@ script:
 - go vet $(go list ./... | grep -v vendor)
 - go test $(go list ./... | grep -v vendor)
 - ./hack/e2e-hostpath.sh
+after_success:
+  - if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+      docker login -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}" quay.io;
+      sudo ./hack/travis-build-all-containers.sh;
+      docker push quay.io/k8scsi/driver-registrar:canary;
```

Comment:
We need a way to configure Canary preferably based on GIT TAG?

Reply:
With so many drivers under one tree it will be hard. I recommend having canary always be master, but having tagged versions of the containers done by hand by the developer.